dom/dom_c.php

```diff
@@ -1312,7 +1312,10 @@ public function removeNamedItem($name) {}
      * in this map).
      */
     #[TentativeType]
-    public function item(#[LanguageLevelTypeAware(['8.0' => 'int'], default: '')] $index): ?DOMNode {}
+    public function item(
+        #[PhpStormStubsElementAvailable(from: '5.3', to: '7.0')] $index = 0,
```

Comment:
No default value for $index? See https://3v4l.org/kYiPo

Reply:
The same case as above. Default value require to mark the parameter as optional